mk/docker.mk

```diff
@@ -49,11 +49,10 @@ image/kuma-prometheus-sd: build/kuma-prometheus-sd/linux-${GOARCH} ## Dev: Rebui
 
 .PHONY: image/kuma-universal
 image/kuma-universal: build/linux-${GOARCH}
-	docker build -t kuma-universal --build-arg ARCH=${GOARCH} --build-arg ENVOY_VERSION=${ENVOY_VERSION}  -f test/dockerfiles/Dockerfile.universal .
```

Comment:
Why doesn't it need to always be tagged as `kuma-universal` anymore?

Reply:
It was tagged twice, we had an image with the tag `latest` and another one with a proper commit tag. New it should be only one